.github/workflows/test.yml

```diff
@@ -19,17 +19,14 @@ jobs:
     name: Pre-Test
     runs-on: ubuntu-latest
 
-    strategy:
-      fail-fast: false
```

Comment:
This `strategy` should have been removed in #12248 when the `matrix` was removed.